mso/resource_mso_remote_location.go

```diff
@@ -120,8 +120,10 @@ func setRemoteLocation(d *schema.ResourceData, remote map[string]interface{}) {
 
 	d.SetId(remote["id"].(string))
 	d.Set("name", remote["name"].(string))
-	d.Set("description", remote["description"].(string))
-
+	description, ok := remote["description"].(string)
```

Comment:
For consistency ->

if description, ok := remote["description"].(string); ok {
		d.Set("description", description)
	}

Reply:
ok